packages/web-styles/src/components/form-range.scss

```diff
@@ -43,3 +44,46 @@
     transform: translateY(#{-0.5 * forms.$form-range-height});
   }
 }
+
+@include utilities.high-contrast-mode() {
+  .form-range {
+    // linear-gradient is forced to the value of "none" in forces-color mode
+    // so, the "forced-color-adjust" property is necessary for "linear-gradient" to continue to work
+    forced-color-adjust: none;
+
+    &:not(:disabled):not(.disabled) {
+      &::-webkit-slider-runnable-track {
+        background: linear-gradient(Highlight, Highlight) 0 / var(--sx) 100%;
+        background-repeat: no-repeat;
+        background-color: ButtonBorder;
+      }
+
+      &::-webkit-slider-thumb {
+        background-color: ButtonFace;
+        border-color: ButtonBorder;
+      }
+
+      &::-moz-range-progress {
+        background: ButtonBorder;
+      }
+
+      &::-moz-range-thumb {
+        background-color: ButtonFace;
+        border-color: ButtonBorder;
+      }
+    }
```

Comment:
Can you add :focus-visible styles for HCM? At the moment I can't see my keyboard focus on the slider. Either an outline on the whole slider or just the thumb, but more pronounced than the shadow in non-HCM.